library/Icinga/Crypt/AesCrypt.php

```diff
@@ -55,12 +55,16 @@ class AesCrypt
 
     const GCM_SUPPORT_VERSION = '7.1';
 
-    public function __construct($random_bytes_len = 128)
+    public function __construct($forceMethod = null, $random_bytes_len = 128)
```

Comment:
I'd like to see a setter here instead.

Reply:
I wanted to do it with setter too, but the problem is, iv and key depend on the method. Or is there a way to call setter before the constructor?

Reply:
Then, instead of throwing a RuntimeError in case of a missing iv and key, generate them on demand in their getter?

Reply:
Sorry, only the iv depends on the method.